pygmt/src/grdclip.py

```diff
@@ -11,6 +10,7 @@
     kwargs_to_strings,
     use_alias,
 )
+from pygmt.io import process_output_grid
```

Comment:
Remember to apply this change elsewhere too!
```suggestion
from pygmt.io import load_datagrid
```